### PR TITLE
fixed the navbar

### LINKED ISSUE #42: fix the navbar

### PR DESCRIPTION
**Pull request by jamespharr**
closes #42

### CODE REVIEW

File: teachme/app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+      <%= link_to 'My Lessons', user_path(current_user) %>
+    <% end if current_user %>
   </div>
   <nav class="navbar fixed-top navbar-dark bg-dark">
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <button type="button" class="btn btn-light"><%= current_user.email if current_user%></button>
+    <%= content_tag :button, class: 'btn btn-light' do %>
+      <%= current_user.email if current_user%>
```
Comment (izzycode) [inline teachme/app/views/layouts/_navbar.html.erb]: here you don't really need the condition since the whole block will only execute on the same condition

Comment (jamespharr) [reply teachme/app/views/layouts/_navbar.html.erb]: I know but I still can see the div with the class btn-light without the email.

Comment (jamespharr) [reply teachme/app/views/layouts/_navbar.html.erb]: Already changed the button anyway!

File: teachme/app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+    <%= content_tag :div, class: 'text-white h4' do %>
+      <%= link_to 'My Lessons', user_path(current_user) %>
+    <% end if current_user %>
   </div>
   <nav class="navbar fixed-top navbar-dark bg-dark">
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <button type="button" class="btn btn-light"><%= current_user.email if current_user%></button>
+    <button class='btn btn-light'> <%= current_user.email if current_user%> </button>
```
Comment (izzycode) [inline teachme/app/views/layouts/_navbar.html.erb]: `<%= content_tag :button current_user.email, class: 'btn btn-light'  if current_user %>

**Pull request merged**